src/app/modules/group/components/group-manager-list/group-manager-list.component.html

```diff
@@ -11,37 +11,80 @@
     [scrollable]="true"
     class="slanted-grid"
     >
-
-    <ng-template #headerTemplate let-columns>
-      <tr style="border: none;">
-        <th style="min-width: 110px;">
-          <div class="slanted-header">
-            <div class="slanted-header-content">
-              Name
+      <ng-template #headerTemplate let-columns>
+        <tr style="border: none;">
+          <th style="width: 160px;">
+            <div class="slanted-header">
+              <div class="slanted-header-content">
+                Name
+              </div>
             </div>
-          </div>
-        </th>
-        <th style="width: 160px; transform: none;">
-      </tr>
-    </ng-template>
-
-    <ng-template
-      #bodyTemplate
-      let-rowData
-      let-columns="columns"
-      let-rowIndex="rowIndex"
-    >
-      <tr [pSelectableRow]="rowData" [pSelectableRowIndex]="rowIndex">
-        <td
-          style="text-align: left;width: 160px;padding-left: 0.5rem;"
-          tooltipPosition="top"
-          tooltipStyleClass="tooltip-custom"
-        >
-          {{ rowData.name }}
-        </td>
+          </th>
+          <th style="min-width: 110px;">
+            <div class="slanted-header">
+              <div class="slanted-header-content">
+                Can Manage
+              </div>
+            </div>
+          </th>
+          <th style="width: 5rem;">
+            <div class="slanted-header">
+              <div class="slanted-header-content">
+                Can grant group access
+              </div>
+            </div>
+          </th>
+          <th style="width: 5rem;">
+            <div class="slanted-header">
+              <div class="slanted-header-content">
+                Can watch members
+              </div>
+            </div>
+          </th>
+          <th style="width: 110px;"></th>
       </tr>
-    </ng-template>
-  </alg-grid>
+      </ng-template>
 
+      <ng-template
+        #bodyTemplate
+        let-rowData
+        let-columns="columns"
+        let-rowIndex="rowIndex"
+      >
+        <tr [pSelectableRow]="rowData" [pSelectableRowIndex]="rowIndex">
+          <td
+            style="text-align: left;width: 160px;padding-left: 0.5rem;"
+            tooltipPosition="top"
+            tooltipStyleClass="tooltip-custom"
+          >
+            {{ rowData.name }}
+          </td>
+          <td style="min-width: 110px;">
+            {{ getManagerLevel(rowData) }}
```

Comment:
The value of the `can_manage` field is now "translated" here to be more user-friendly

Reply:
You cannot do it this way, otherwise this function will be called hundreds times. No function calls can be done in the rendering. 
One way to solve that is after `.getGroupManagers(this.group.id)`  to map each managers to a dictionary where you have added a "can_manage_as_text" for instance. And so having `managers: Manager[] = [];` -> `managers: (Manager&{can_manage_as_text:string})[] = [];` for instance.
You can check what I did with functions such as "withCodeAdditions" but here it is just something internal to this class, it can probably be done more easily.

Reply:
Ok I fixed it (I didn't know how to do this properly).